docs/source/user-guide/general/metrics_guide.rst

```diff
@@ -259,12 +261,29 @@ Custom Metrics
 ==============
 You can also pass your own custom metric to relevant checks and suites.
 
-For computer vision the custom metrics should support the
-`Ignite Metric <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__ API.
-
-For tabular metrics the custom metrics should support the
-`sklearn scorer <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html>`__ API.
-Multiclass classification scorers should assume that the labels are given in a
+For tabular metrics and vision classification tasks the custom metrics function should follow the
+`sklearn scorer <https://scikit-learn.org/stable/modules/generated/sklearn.metrics.make_scorer.html>`__ API, which is
+a function that accepts the parameters: (model, x, y_true), and returns a score with the convention that higher is
+better.
+
+For other computer vision tasks, the custom metrics should be a custom metric class. Deepchecks Custom Metric classes
+are inspired by `Ignite metrics <https://pytorch.org/ignite/metrics.html#how-to-create-a-custom-metric>`__, must
+inherit from :class:`deepchecks.vision.metric_utils.custom_metric.CustomMetric` and implement the following methods:
+``reset``, ``update`` and ``compute``.
+
+    * ``reset`` - resets the metric to its initial state, creates or resets any internal variables.
+    * ``update`` - updates the metric's internal state based on the provided labels and predictions. The method's
+        signature should be ``update(self, output)``, where output is a tuple containing first ``y_pred`` which is the
```

Comment:
thats how ignite metrics work? otherwise I think update(self, y_true,y_pred) is better as convention

Reply:
yep that's how ignite works

Reply:
:(